src/test/java/fi/fmi/avi/model/swx/amd82/immutable/SpaceWeatherAdvisoryAmd82Test.java

```diff
@@ -55,14 +54,23 @@ private NextAdvisory getNextAdvisory(final boolean hasNext) {
         return next.build();
     }
 
-    private List<String> getRemarks() {
+    private static List<String> getRemarks() {
         final List<String> remarks = new ArrayList<>();
         remarks.add("RADIATION LVL EXCEEDED 100 PCT OF BACKGROUND LVL AT FL350 AND ABV. THE CURRENT EVENT HAS PEAKED AND LVL SLW RTN TO BACKGROUND LVL."
                 + " SEE WWW.SPACEWEATHERPROVIDER.WEB");
 
         return remarks;
     }
 
+    private static List<AdvisoryNumber> getReplacementNumbers(final int... serials) {
+        return Arrays.stream(serials)
+                .mapToObj(serial -> AdvisoryNumberImpl.builder()
+                        .setYear(2020)
+                        .setSerialNumber(serial)
+                        .build())
+                .collect(ImmutableList.toImmutableList());
+    }
```

Comment:
Same here about using Guava. Here returning the plain `Stream` without collecting would probably be the most efficient solution?